src/main/scala/mesosphere/marathon/core/scheduler/SchedulerPluginConfigurer.scala

```diff
@@ -0,0 +1,27 @@
+package mesosphere.marathon
+package core.scheduler
+
+import javax.inject.Inject
+
+import mesosphere.marathon.core.plugin.PluginManager
+import mesosphere.marathon.plugin.scheduler.SchedulerPlugin
+import org.slf4j.LoggerFactory
+
+/**
+  * Workaround to integrate [[mesosphere.marathon.core.scheduler.SchedulerPluginRunner]] (object)
+  * with [[mesosphere.marathon.core.plugin.PluginManager]] (created with Guice)
+  *
+  * @author Dmitry Tsydzik.
+  * @since 6/15/17.
+  */
+class SchedulerPluginConfigurer @Inject() (pluginManager: PluginManager) {
+  private[this] val log = LoggerFactory.getLogger(getClass)
+
+  try {
+    SchedulerPluginRunner.plugins = pluginManager.plugins[SchedulerPlugin].toList
+  } catch {
+    case e: Throwable =>
+      log.error("Failed to load plugins", e)
```

Comment:
is it really worth using `error` level logging if the plan is to rethrow the exception? maybe `debug`  or `info` level would be more appropriate?